control-plane/api-gateway/common/translation.go

```diff
@@ -141,6 +147,47 @@ func ToContainerPort(portNumber gwv1beta1.PortNumber, mapPrivilegedContainerPort
 	return int(portNumber) + int(mapPrivilegedContainerPorts)
 }
 
+func (t ResourceTranslator) translateGatewayPolicy(policy *v1alpha1.GatewayPolicy) (*api.APIGatewayPolicy, *api.APIGatewayPolicy) {
+	if policy == nil {
+		return nil, nil
+	}
+
+	defaultPolicy := &api.APIGatewayPolicy{
+		JWT: t.translateJWTRequirement(policy.Spec.Default.JWT),
```

Comment:
I think we need a `nil` check on both `policy.Spec.Default` and `policy.Spec.Override` since both are optional